src/test/java/test/domain/LottoTest.java

```diff
@@ -36,6 +38,49 @@ class LottoTest {
         assertThrows(IllegalArgumentException.class, () -> new Lotto(duplication));
     }
 
+    @Test
+    void 보너스번호가_포함되면_true를_반환한다() {
+        Lotto lotto = new Lotto(Arrays.asList(1, 2, 3, 4, 5, 6));
+        BonusNumber bonus = new BonusNumber(6);
+        assertTrue(lotto.isBonusNumberMatched(bonus));
+    }
+
+    @Test
+    void 보너스번호가_포함되지않으면_false를_반환한다() {
+        Lotto lotto = new Lotto(Arrays.asList(1, 2, 3, 4, 5, 6));
+        BonusNumber bonus = new BonusNumber(7);
+        assertFalse(lotto.isBonusNumberMatched(bonus));
+    }
```

Comment:
요거 같은 경우엔 `isBonusNumberMatched(...)`를 사용하므로 `@Nested`를 사용해서 계층화가 가능해보여요~

이거는 처음에 테스트 코드 생성할 때는 큰 의미는 없지만, 나중에 테스트 코드가 더 추가되거나, 기존 코드 수정할 때 분류가 되어서 편하더라구요

```suggestion
    @Nested
    class isBonusNumberMatched_테스트 {

        @Test
        void 보너스번호가_포함되면_true를_반환한다() {
            Lotto lotto = new Lotto(Arrays.asList(1, 2, 3, 4, 5, 6));
            BonusNumber bonus = new BonusNumber(6);
            assertTrue(lotto.isBonusNumberMatched(bonus));
        }

        @Test
        void 보너스번호가_포함되지않으면_false를_반환한다() {
            Lotto lotto = new Lotto(Arrays.asList(1, 2, 3, 4, 5, 6));
            BonusNumber bonus = new BonusNumber(7);
            assertFalse(lotto.isBonusNumberMatched(bonus));
        }
    }
```

Reply:
"계층화" 라는 용어가 생소해서 찾아보니 기능별로 묶는다는 뜻이었군요!
안그래도 로직별로 테스트가 있는 `lotto` 객체의 경우 로직별로 구별하면 깔끔하겠다는 생각을 내심 하고 있었습니다.
해당 방식으로 수정했습니다!